src/mantarray_file_manager/file_writer.py

```diff
@@ -4,12 +4,16 @@
 import ntpath
 import os
 from os import getcwd
+from typing import Any
 from typing import Optional
 from typing import Tuple
 from typing import Union
 import uuid
 
 import h5py
+from immutable_data_validation.wrapped_vc_validators import validate_int
```

Comment:
note for future: if you check the `__init__.py` file of a package, you can see what can be directly imported from the package itself, e.g. `from immutable_data_validation import validate_int` 

As long as the namespace doesn't get too cluttered, this provides a way to avoid causing problems to packages using a library when reorganization/refactoring is done inside a package.  For example, if we decided to rename the `wrapped_vc_validators` module to something else, or move `validate_int` to a different module, your current code would break, but not the example above

https://github.com/CuriBio/immutable-data-validation/blob/e9870ac4ad25f049d6d4494e7127c3d1839705de/src/immutable_data_validation/__init__.py#L17